contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -230,27 +208,7 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         _unstake(validatorId, currentEpoch);
     }
 
-    // Housekeeping function. @todo remove later
-    // other than varibale you want to update give same values
-    function updateConstructor(
```

Comment:
Shouldn't we keep this? Otherwise how we are going to change `_stakingLogger` or `_rootchain`?